Documentation/ceph-cluster-crd.md

```diff
@@ -131,18 +130,18 @@ Below are the settings available, both at the cluster and individual node level,
 
 - `useAllDevices`: `true` or `false`, indicating whether all devices found on nodes in the cluster should be automatically consumed by OSDs. **Not recommended** unless you have a very controlled environment where you will not risk formatting of devices with existing data. When `true`, all devices will be used except those with partitions created or a local filesystem. Is overridden by `deviceFilter` if specified.
 - `deviceFilter`: A regular expression that allows selection of devices to be consumed by OSDs.  If individual devices have been specified for a node then this filter will be ignored.  This field uses [golang regular expression syntax](https://golang.org/pkg/regexp/syntax/). For example:
-  - `sdb`: Only selects the `sdb` device if found
-  - `^sd.`: Selects all devices starting with `sd`
-  - `^sd[a-d]`: Selects devices starting with `sda`, `sdb`, `sdc`, and `sdd` if found
-  - `^s`: Selects all devices that start with `s`
-  - `^[^r]`: Selects all devices that do *not* start with `r`
+  - `sdb`: Only selects the `sdb` device if found.
```

Comment:
The trend I see in most of the rest of the docs and the guideline I see most often is to make sure bulleted list items do *not* end in a period.